crates/resolver/src/error.rs

```diff
@@ -91,11 +93,17 @@ impl ResolveError {
 
 impl fmt::Display for ResolveError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        if let Some(ref backtrace) = self.backtrack {
-            fmt::Display::fmt(&self.kind, f)?;
-            fmt::Debug::fmt(backtrace, f)
-        } else {
-            fmt::Display::fmt(&self.kind, f)
+        cfg_if::cfg_if! {
+            if #[cfg(feature = "with-backtrace")] {
```

Comment:
Should this (and other instances) be changed to just `"backtrace"`?

Reply:
I guess so? The current change seems incoherent. Is that wrong on main as well?

Reply:
Yes, that is what currently exists on main--

https://github.com/bluejekyll/trust-dns/blob/ac8bc7fa03f0ddd36762079e9e25cbb3023faf2d/crates/resolver/src/error.rs#L253

Reply:
Addressed in commit https://github.com/bluejekyll/trust-dns/pull/1387/commits/29633fe2ae968a80e3727a8de386fd30d49c8f5b, everything should now be using the `"backtrace"` feature.